config/config.toml

```diff
@@ -38,7 +51,7 @@ password = "default-password"
 
 [services.did]
 name = "did"
-methods = ["key", "web", "ion"]
+methods = ["key", "web"]
```

Comment:
Curious why this change?

Reply:
having ion as a method requires a write node, and until ours is a bit more stable would like to avoid putting it here for everyone to use

Reply:
This file is the one used for local development. I don't follow why everyone would use it if we leave this in here. 

I'm also confused by why the values in this field require a write node. For instance "web" doesn't allow any writes, since there is no way or programmatically writing a did:web DID. Can you elaborate?

Reply:
why does local deployment need a dependency on an ion write node?

not all DID methods require similar infra. we could refactor to separate out creating from anchoring...maybe in #340